back-end/service.py

```diff
@@ -87,38 +90,28 @@ def designWorkflow(files, repoPath):
 
 	print("Workflow written at " + repoPath + "/Snakefile")
 
-def getAnnexedFiles(files, repoPath):
-	lines = []
-	with open('../templates/drone.template', 'r+') as template:
-		for line in template.readlines():
-			if not '#get-annexed-files' in line: lines.append(line)
-			else:
-				input_files = ""
-				for filename in files.values():
-					input_files += "'{}' ".format(filename)
-				lines.append(("    #- git annex get {}").format(input_files))
-
-	template.close()
-
-	with open(repoPath + '/.drone.yml', 'w+') as config: config.writelines(lines)
-	config.close()
-
-	print("Annexed files pulling configuration added at " + repoPath + "/.drone.yml")
 
-def designBackPush(files, repoPath):
+def designCIConfig(backPushfiles, annexFiles, repoPath):
 	lines = []
 	with open('../templates/drone.template', 'r+') as template:
 		for line in template.readlines():
 			if '#add-backpush-files' in line:
```

Comment:
[Detected Error] Build fails at this point if no backpush file is specified by the user. Ideally it should skip the entire process of adding the new dir and pushing it.